rust/src/tx_builder.rs

```diff
@@ -389,19 +393,19 @@ impl TransactionBuilder {
 
     pub fn build(&self) -> Result<TransactionBody, JsError> {
         let fee = self.fee.ok_or_else(|| JsError::from_str("Fee not specified"))?;
-        let ttl = self.ttl.ok_or_else(|| JsError::from_str("ttl not specified"))?;
```

Comment:
Since it's no longer mandatory I didn't put it here even though the field is still required if you created it through the `TransactionBody::new()` field. I suppose we could make it so that either this is mandatory here or change the constructor to be optional?

Reply:
Yeah I think removing it is fine.